extensions/pom.xml

```diff
@@ -25,6 +25,7 @@
         <module>all-extensions</module>
         <module>file</module>
         <module>ftp</module>
+        <module>http</module>
```

Comment:
do you want this to be part of the distribution? if so you're missing the entire assembly, whitelist, extensions-all piece